src/catalog/src/builtin.rs

```diff
@@ -4054,6 +4071,30 @@ WHERE worker_id = 0",
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_DATAFLOW_GLOBAL_IDS: LazyLock<BuiltinView> = LazyLock::new(|| BuiltinView {
+    name: "mz_dataflow_globalids",
```

Comment:
```suggestion
    name: "mz_dataflow_global_ids",
```

For consistency with the column name.